test_red_black_cards.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+from red_black_cards import expected_value
```

Comment:
According to Mo, avoid statements like `from foo import bar`. It's better to use the fully qualified function name like `foo.bar` so it's easier to read.